components/Dialogs/src/MDCAlertController.m

```diff
@@ -465,6 +465,20 @@ - (void)viewWillTransitionToSize:(CGSize)size
                       completion:nil];
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+  [super viewDidAppear:animated];
+
+  if (self.alertView.contentScrollView.contentSize.height >
+      CGRectGetHeight(self.alertView.contentScrollView.bounds)) {
+    [self.alertView.contentScrollView flashScrollIndicators];
+  }
+
+  if (self.alertView.actionsScrollView.contentSize.height >
+      CGRectGetHeight(self.alertView.actionsScrollView.bounds)) {
+    [self.alertView.actionsScrollView flashScrollIndicators];
+  }
```

Comment:
I think you don't need these checks; you should just be able to flash the scroll indicators and UIKit will determine whether it needs to.

Reply:
Good call! Updated the PR.